templates/pkg/resource/references.go.tpl

```diff
@@ -143,6 +143,17 @@ func resolveReferenceFor{{ $field.FieldPathWithUnderscore }}(
 	}
 	return nil
 }
+{{ else if not $isList -}}
+//is nested
```

Comment:
Please use a Go template comment instead of a Go comment:

```
{{/* nested field */}}
```

Reply:
Done